src/notyf/toast-container/toast-container.component.ts

```diff
@@ -1,8 +1,8 @@
 import { Component, OnInit } from '@angular/core';
 
 @Component({
-    selector: 'app-toast-container',
-    template: ``,
+    selector: 'notyf-toast-container',
+    template: `<div></div>`,
```

Comment:
I think this `div` element is not necessary, check the `DOM` after adding an element, it will be there just hanging 😋